server/src/service.rs

```diff
@@ -151,10 +168,21 @@ impl Service {
 
     async fn fetch_collections(&self) -> Vec<Collection> {
         let mut collections = Vec::new();
-        // todo: create default collection
-
+        // create login collection
+        let login_collection = Collection::new(
```

Comment:
The keyring should be associated to the collection, what you are doing here is wrong. You canno't associate the login collection to the same keyring as the session collection

Reply:
Yes, you're right. I have removed this, thanks!